src/App.jsx

```diff
@@ -14,33 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortingElements(goods, sortField, isReversed) {
+  const sortedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (sortField) {
+    case 'alphabetic':
+      sortedGoods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
+    case 'length':
+      sortedGoods.sort((good1, good2) => good1.length - good2.length);
+      break;
+    case 'reset':
+      return goodsFromServer;
+    default:
+      break;
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return isReversed ? sortedGoods.reverse() : sortedGoods;
+}
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+
+  const goods = sortingElements(goodsFromServer, sortField, isReversed);
+
+  const handleSort = field => {
+    if (field === 'reverse') {
+      setIsReversed(!isReversed);
+    } else {
+      setSortField(field);
+    }
+  };
+
+  const handleReset = () => {
+    setSortField('');
```

Comment:
The `handleReset` function sets `sortField` to an empty string. This does not match any case in the `sortingElements` function, which could lead to unexpected behavior. Consider using a specific value like 'reset' to align with the existing case in the `sortingElements` function.